rdkit/Chem/UnitTestCatalog.py

```diff
@@ -14,7 +14,7 @@
 from rdkit import RDConfig
 from rdkit import Chem
 from rdkit.Chem import FragmentCatalog, BuildFragmentCatalog
-from rdkit.six.moves import cPickle
+from rdkit.six.moves import cPickle  # @UnresolvedImport
```

Comment:
I'm not going to stop accepting the PR, but I really hate these artifacts getting dropped in the Python code just to stop things like coverage checkers and linters from complaining.